### ISSUE TITLE
✅ 🐞 Bug report: When template Section is added into a Page, it has trouble loading responsive styles

### ISSUE THREAD

**Issue by aureliePrd** [elementor/elementor#18412]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When I create a Section Template and insert it into a page, the responsive styles for this section are not loading properly. It gets fixed if I regenerate the CSS and files in Elementor / Tools but it got back wrong again if I open and modify the section template.

I have noticed this issue for a while now (several months)

### Steps to reproduce

1. Create a section template
2. add some desktop styles (ex: 3px border, columns align center, title align center) and some responsive styles (ex: 1px border, columns align space-between, title align left)
3. insert this section into a regular elementor page
4. save and load the page in front
5. change something in the section template (even a color)
6. save and load the page in front --> the responsive style is not added
7. regenerate files and css and reload the page --> should be OK
8. repeat from step 5.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MySQL Community Server (GPL) v5.7.30
	PHP Version: 7.4.28
	PHP Memory Limit: 512M
	PHP Max Input Vars: 3000
	PHP Max Post Size: 300M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.9.3
	Site URL: https://map-dev.ch/piscinedelavenoge
	Home URL: https://map-dev.ch/piscinedelavenoge
	WP Multisite: No
	Max Upload Size: 300 Mo
	Memory limit: 640M
	Max Memory limit: 640M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: fr-FR
	Timezone: Europe/Paris
	Debug Mode: Inactive

== Theme ==
	Name: Agence MAP - Piscine de la Venoge - Theme Enfant de Hello Elementor
	Version: 1.0.1
	Author: MAP Team - Aurélie Perrard
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.5.0
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: fr_FR
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.75 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields
		Version: 5.12.2
		Author: Delicious Brains

	Elementor
		Version: 3.6.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.6.4
		Author: Elementor.com


== Utilisation des éléments ==
	
	header : 2
		heading : 2
		template : 2
	section : 2
		button : 2
		heading : 2
		nav-menu : 1
		theme-site-logo : 1
	wp-post : 0
		button : 2
		heading : 5
		nav-menu : 1
		template : 1
		text-editor : 2
		theme-page-title : 1
		theme-site-logo : 1
	wp-page : 2
		heading : 2
		template : 1
		text-editor : 1
		theme-page-title : 1


== Expériences d’Elementor ==
	Sortie DOM optimisée: Actif par défaut
	Amélioration du chargement des ressources: Actif par défaut
	Optimisation du chargement du CSS: Actif par défaut
	Inline Font Icons: Inactif par défaut
	Améliorations de l’accessibilité: Actif par défaut
	Kit de modèles pour importation et exportation: Actif par défaut
	Points d’arrêt personnalisés supplémentaires: Actif par défaut
	Masquer les widgets natifs de WordPress des résultats de recherche: Actif par défaut
	: Inactif par défaut
	Container Flexbox: Inactif par défaut
	Default to New Theme Builder: Actif par défaut
	En-tête et pied de page du thème Hello: Actif par défaut
	Page d’atterrissage: Actif par défaut
	Echantillon de couleurs: Actif par défaut
	Widgets favoris: Actif par défaut
	Barre supérieure d’administration: Actif par défaut
	Page Transitions: Actif par défaut
	Envois de formulaire: Actif par défaut
	Scroll Snap: Actif par défaut


== Journal ==
	
PHP: showing 2 of 2PHP: 2022-04-07 09:56:44 [notice X 2][../wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-04-07 10:45:51 [warning X 1][../wp-content/plugins/elementor/core/files/file-types/svg.php::233] simplexml_load_file(): I/O warning : failed to load external entity "https://map-dev.ch/piscinedelavenoge/wp-content/uploads/2022/04/logo_aipcv.svg" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 6 of 6JS: 2022-04-07 09:08:15 [error X 1][https://map-dev.ch/piscinedelavenoge/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: \"view26283\") has already been destroyed and cannot be used. 
JS: 2022-04-07 10:22:00 [error X 8][https://map-dev.ch/piscinedelavenoge/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: \"view85\") has already been destroyed and cannot be used. 
JS: 2022-04-07 12:13:53 [error X 1][https://map-dev.ch/piscinedelavenoge/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: \"view1117\") has already been destroyed and cannot be used. 
JS: 2022-04-11 07:17:09 [error X 2][https://map-dev.ch/piscinedelavenoge/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: \"view121\") has already been destroyed and cannot be used. 
JS: 2022-04-11 07:25:15 [error X 1][https://map-dev.ch/piscinedelavenoge/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: \"view2548\") has already been destroyed and cannot be used. 
JS: 2022-04-11 07:29:31 [error X 6][https://map-dev.ch/piscinedelavenoge/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: \"view142\") has already been destroyed and cannot be used. 

Log: showing 5 of 52022-04-11 09:29:12 [info] ae-pro::aepro_updater Started 
2022-04-11 09:29:12 [info] AnyWhere Elementor Pro/Upgrades - _v_2_21_0 Start  
2022-04-11 09:29:12 [info] AnyWhere Elementor Pro/Upgrades - _v_2_21_0 Finished 
2022-04-11 09:29:12 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'AnyWhere Elementor Pro',
  'from' => '0.1',
  'to' => '2.24',
)]
2022-04-11 09:29:12 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'AnyWhere Elementor Pro',
  'from' => false,
  'to' => '2.24',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
	

				

````

</details>

**Comment by eastriverstudio** [comment]
I can confirm I'm also getting this same/similar issue when using theme builder templates. In my case I have a default page template and I noticed when editing any responsive styles of a page using this template look fine in the editor and initially OK on front-end immediately after I save the page from the Elementor editor, but if anyone or myself goes to edit the page afterwards, the responsive styles particularly on sections and columns seem to removed. Regenerating the css in Elementor settings usually fixes it temporarily but it reverts back once the page is editing in any way. 

Looking in to the css which is printed for the page, the styles added inline are overriding or missing the correct responsive styling, although it is present in the separate css file which also gets loaded. Looks like it is some kind of issue related to the inline css experiments. I've tried turning those off too and the inline css is still printed to the page without the correct responsive styling.

**Comment by eastriverstudio** [comment]
@aureliePrd see update on https://github.com/elementor/elementor/issues/16467 it's a different use of templates, but maybe there's a similar issue using templates generally and the same workaround could help?

**Comment by gregolias** [comment]
As a workaround, you need to click on the “Preview” button while editing your footer HTML block. And always keep the preview window opened whiled customizing and saving your HTML blocks with Elementor.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Core 3.15.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
